.ci_support/build_notebooks_damask.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+for f in $(cat .ci_support/exclude_damask); do 
```

Comment:
I would suggest naming this `exclude_for_damask` instead, and similarly `exclude_for_fenics` to (help) avoid the same confusion Sam had.